cloud/amazon/ec2_elb_lb.py

```diff
@@ -698,6 +710,28 @@ def _set_cross_az_load_balancing(self):
         self.elb_conn.modify_lb_attribute(self.name, 'CrossZoneLoadBalancing',
                                           attributes.cross_zone_load_balancing.enabled)
 
+    def _set_access_log(self):
+        attributes = self.elb.get_attributes()
+        if self.access_logs is not None:
+            access_log = boto.ec2.elb.attributes.AccessLogAttribute()
+            access_log.enabled = True
+            access_log.s3_bucket_prefix = ''
+            access_log.emit_interval = 60
+            if 's3_location' not in self.access_logs:
+              self.module.fail_json(msg='s3_location information required')
+            access_log.s3_bucket_name = self.access_logs['s3_location']
+            if 's3_prefix' in self.access_logs:
+              access_log.s3_bucket_prefix = self.access_logs['s3_prefix']
+            if 'interval' in self.access_logs:
+              access_log.emit_interval = self.access_logs['interval']
+            if str(access_log) != str(attributes.access_log):
```

Comment:
Thanks for looking at my code...
There is no good way to compare those objects unfortunately:
```
>>> dir(AccessLogAttribute())
['__class__', '__delattr__', '__dict__', '__doc__', '__format__', '__getattribute__', '__hash__', '__init__', '__module__', '__new__', '__reduce__', '__reduce_ex__', '__repr__', '__setattr__', '__sizeof__', '__str__', '__subclasshook__', '__weakref__', 'emit_interval', 'enabled', 'endElement', 's3_bucket_name', 's3_bucket_prefix', 'startElement']
>>> a = AccessLogAttribute()
>>> b = AccessLogAttribute()
>>> a
AccessLog(None, None, None, None)
>>> b
AccessLog(None, None, None, None)
>>> a == b
False
```

Following your advices, I changed the code to use `__dict()__` instead... let me know.